build/buildconstants/params_calibnet.go

```diff
@@ -113,6 +110,11 @@ const UpgradeTeepHeight = 9999999999
 // ramp behavior before mainnet upgrade.
 var UpgradeTuktukPowerRampDurationEpochs = uint64(builtin.EpochsInDay * 3)
 
+// Canceled - See update in: https://github.com/filecoin-project/community/discussions/74#discussioncomment-11549619
+const UpgradeTeepHeight = 9999999999
+
+var UpgradeTeepInitialFilReserved = wholeToNano(1_100_000_000) // FIP-0100: 300M -> 1.1B FIL
```

Comment:
```suggestion
var UpgradeTeepInitialFilReserved = wholeToNano(900_000_000) // FIP-0100: 300M -> 900M FIL
```

This would match the current FIP. Need to consider whether it's better to be correct, or make it closer to mainnet just to test fee calculation. 🤔

Reply:
I think one of the goals for Calibnet is to try to mirror Mainnet. While it has achieved that with regards to SP hardware usage, block times, and proofs, the economics has been really skewed.

I'm in favor of aligning Calibnet more closely with Mainnet's economic conditions. Although we may never fully replicate Mainnet econ due to differences in scale and miner numbers, adjusting the circulating supply to be more accurate to what Mainnet is a positive step forward in my opinion.

Reply:
https://github.com/filecoin-project/FIPs/pull/1133 & https://github.com/filecoin-project/FIPs/discussions/1105

Proposing another 100M FIL to get us to an almost exact match with mainnet. I'm still not sure about it, but these really are just test networks so we can tweak them as we need and it would be nice to have equations resolving to very similar values as mainnet.